Dockerfile

```
@@ -56,7 +56,7 @@ RUN curl -sSL https://nixpacks.com/install.sh -o install.sh \
     && pnpm install -g tsx
 
 # Install Railpack
-ARG RAILPACK_VERSION=0.0.64
+ARG RAILPACK_VERSION=0.0.66
```

```suggestion
ARG RAILPACK_VERSION=0.0.64
```

I don't want to update the railpack since can introduce breaking changes, I need a better wait to figure out since this can break deploy for remote servers, so for now let's revert